server/portal/templates/includes/nav_cms.html

```diff
@@ -0,0 +1,12 @@
+{# @var className #}
+
+{# RFE: Use a custom element or React component whose code exists in one repo #}
+{# WARNING: Some markup is duplicated in other repositories #}
+{# SEE: https://confluence.tacc.utexas.edu/x/LoCnCQ #}
+<ul class="s-cms-nav  {{ className }}" id="cms-nav-pages" data-cms-nav-endpoint="/cms/nav/pages/markup/">
+
+  <!-- All top-level CMS controlled nav elements handled here. -->
+  {# FAQ: Content populated via JavaScript, below #}
+
+</ul>
+<script src="/static/site_cms/js/nav_cms.js"></script>
```

Comment:
__Caveat__: This implementation is not as explicit as it should be, but it is a stopgap (and some future improvements are commented in `nav_cms.js`). I'm saving a full-fledged plan for _after_ [Site Search](https://jira.tacc.utexas.edu/browse/FP-208) is implemented, which _could_ support a React component in the CMS. _There is a [WIP (oversized) PoC](https://gitlab.tacc.utexas.edu/wma-cms/texascale-org/compare/master...develop--task%2FGL-31-convert-image-map-to-component) of this idea._